### PR TITLE
Milestone: Refactor to use JavaScript classes

### PR DESCRIPTION
**Pull request by newhorizon-tech**
Features 

- Display Books
- Add Books 
- Remove Books
- Store Books in LocalStorage
-  Plain Javascript 
-  Use Classes

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,5 +1,28 @@
 let bookList = [];
```
Comment (Lameck1) [inline index.js]: - [x] We are allowed to have arrays and objects, but they should all be in a class. Consider refactoring this into the Book class as a static property of the class, or go the route of two classes, one for booksCollection and the other defining a Book Object.

File: index.js [hunk 2]
```diff
@@ -1,5 +1,28 @@
...
+    bookList.push(book);
+  }
+
+  removeBook() {
+    bookList = bookList.filter((book) => {
+      const titleMatch = book.title === this.title;
+      const authorMatch = book.author === this.author;
+      return !(titleMatch && authorMatch);
+    });
+  }
```
Comment (Lameck1) [inline index.js]: - [x] This allows the entry of duplicate books. The problem when a user tries to delete one of the duplicates, all other books with the same titles also get deleted. We are only allowed to delete one specific book at a time. Look at the quote below from the project requirements.

## From the previous milestone

> - Make sure that when a user clicks the "Remove" button:
>   - The correct book is removed from the collection.
>   - The correct book dissapears from the page.

## In this milestone, we are now expected to maintain the above functionality.

> Maintain all of the original functionality of the application (i.e. no bugs were introduced while refactoring).

Please work on this. You can choose to filter Books by a unique ID or prevent the addition of duplicate books.

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi @newhorizon-tech , 

Splendid job so far!
We still have some issues to iron out before we can move ahead though.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by newhorizon-tech** [comment]
Changes made 

- Used two classes (One for Book and another for BookCollection) 
- Website now prevents user from adding duplicate books

**Review by smcommits — APPROVED** [review]
### Status: Approved 👍🏽 

Hi @newhorizon-tech, 

## You have done a great job! Your project is complete. This PR is now approved. You can go ahead and merge the branch. Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**